server/taskflows/hpccloud/taskflow/nwchem/neb.py

```diff
@@ -0,0 +1,54 @@
+import cumulus.taskflow.cluster
+from cumulus.taskflow.cluster import create_girder_client
+from cumulus.tasks.job import submit_job, _monitor_jobs
+from cumulus.tasks.job import download_job_input_folders
+from cumulus.tasks.job import upload_job_output_to_folder, job_directory
+from cumulus.transport import get_connection
+from . import setup_input, create_job, submit, submit_nwchem_job, \
+    monitor_nwchem_job, upload_output
+
+
+from hpccloud.taskflow.utility import *
+
+import sys
+
+class NWChemTaskFlow(cumulus.taskflow.cluster.ClusterProvisioningTaskFlow):
```

Comment:
I am a little confused. This class appears to be defined twice. Here and in the new [__init__.py](https://github.com/Kitware/HPCCloud/blob/nwchem-neb/server/taskflows/hpccloud/taskflow/nwchem/__init__.py#L33)